server/src/main/java/org/candlepin/controller/OwnerManager.java

```diff
@@ -118,7 +118,7 @@ public void cleanupAndDelete(Owner owner, boolean revokeCerts) {
 
         for (Consumer consumer : consumers) {
             log.info("Removing all entitlements for consumer: {}", consumer);
-            poolManager.revokeAllEntitlements(consumer, revokeCerts);
+            poolManager.revokeAllEntitlements(consumer, false);
```

Comment:
In the cases where we've made a concious decision to set true/false when revoking all ents, do you think it would make sense to add a comment as to why this is? Just a thought, and no worries if you don't agree. Just think it would be easier to understand when reading the code without having to go back to the commit later.

Reply:
The only argument I would have against more comments is avoiding pushing another change that will re-trigger the tests.

I can add a quick note to each of these stating that we're deleting the consumer anyway, so there's no value in regenerating entitlements and status.